### PR TITLE
Remove obsolete `WordPressFlux.xcodeproj`

### PR DESCRIPTION
**Pull request by mokagio**
The project is unnecessary since we have moved to package WordPressFlux as an internal Swift Package.

Besides, if one opened the project, they would have seen a lot of dead references and a failed build. 

Screen Shot 2022-06-16 at 3 11 29 pm

Screen Shot 2022-06-16 at 3 11 49 pm

## Testing 

If CI is green, then WordPressFlux hasn't been compromised by this change.

## Regression Notes

1. Potential unintended areas of impact – N.A.
2. What I did to test those areas of impact (or what existing automated tests I relied on) – N.A.
3. What automated tests I added (or what prevented me from doing so) – N.A.

---

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes. **N.A.**
- [x] I have considered adding accessibility improvements for my changes. **N.A.**
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. **N.A.**

### CODE REVIEW

**Review by AliSoftware — COMMENTED** [review]
Doing a silver search `ag WordPressFlux` in the repo I found some other references to this project that should likely be deleted alongside this, to avoid orphan references.

⚠️  This scheme seems to reference the `.xcodeproj` that you're about to delete in this PR:
```
WordPress/WordPress.xcodeproj/xcshareddata/xcschemes/WordPress.xcscheme
108:               BuildableName = "WordPressFluxTests.xctest"
109:               BlueprintName = "WordPressFluxTests"
110:               ReferencedContainer = "container:../WordPressFlux/WordPressFlux.xcodeproj">
```

❔ This xctestplan references WordPressFlux, but I think it's referencing the Swift Package, not the project (the path of the container doesn't have `.xcodeproj` suffix, after all) so maybe that's a false positive and this one is all good…
```
WordPress/WordPressTest/WordPressUnitTests.xctestplan
37:        "containerPath" : "container:..\/WordPressFlux",
38:        "identifier" : "WordPressFluxTests",
39:        "name" : "WordPressFluxTests"
```

❔ Same with this, I think it might be referencing the Swift Package not the xcodeproj, so I think we're ok, but could be worth double-checking
```
WordPress.xcworkspace/contents.xcworkspacedata
8:      location = "group:WordPressFlux">
```

**Comment by mokagio** [comment]
@AliSoftware thank you for silver searching 🙇‍♂️ 

I removed the dead reference in the scheme and double checked the other ones were legit. 

In doing so, I noticed other dead references in the WordPress scheme, see all the "(missing)" in the figure below:

Screen Shot 2022-06-20 at 9 12 49 pm

I'll address them in a dedicated followup PR to keep this one focused. _Update: Done in https://github.com/wordpress-mobile/WordPress-iOS/pull/18915._

File: WordPressFlux/.swiftpm/xcode/xcshareddata/xcschemes/WordPressFlux.xcscheme [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      </MacroExpansion>
+   </ProfileAction>
+   <AnalyzeAction
+      buildConfiguration = "Debug">
+   </AnalyzeAction>
+   <ArchiveAction
+      buildConfiguration = "Release"
+      revealArchiveInOrganizer = "YES">
+   </ArchiveAction>
+</Scheme>
```
Comment (mokagio) [inline WordPressFlux/.swiftpm/xcode/xcshareddata/xcschemes/WordPressFlux.xcscheme:67]: I originally didn't include this, but Xcode keeps generate the `.swiftpm` folder when I change the WordPress scheme. 

The project builds fine without this, but if there's ever a time to track it, it is now, to save noise in other PRs.

Comment (AliSoftware) [reply WordPressFlux/.swiftpm/xcode/xcshareddata/xcschemes/WordPressFlux.xcscheme:67]: TIL that SwiftPM generates xcschemes for Xcode

**Review by AliSoftware — APPROVED** [review]

**Pull request merged**